osmtm/version.py

```diff
@@ -0,0 +1,2 @@
+from gitversion import determine_git_version
+__VERSION__ = determine_git_version(__file__)
```

Comment:
@pyrog can you please remove the `version.py` file. I'd be happy to merge it then. Thanks.